birdhouse/components/weaver/docker-compose-extra.yml

```diff
@@ -114,6 +114,19 @@ services:
       timeout: 10s
       retries: 2
 
+  # Dedicated database for Weaver, since other 'mongodb' image does not employ the same version.
+  weaver-mongodb:
+    image: mongo:5.0
```

Comment:
Want to put that mongo image version into weaver/default.env as well so it is configurable and follow the same convention as weaver.  Is that a specific version or a "rolling" version?  Let's use a specific version for reproductibility.

Reply:
I'm not sure what you mean by rolling version, but it is a minimum requirement for Weaver.
I didn't add a default.env parameter for this reason.

Reply:
On https://hub.docker.com/_/mongo?tab=tags, currently there is `5.0.4`,  `5.0` and `5`.  I suspect `5.0` and `5` are rolling tag (like the `latest` tag) and only `5.0.4` is a specific tag that will not change with time.

For reproductibility, we want to avoid rolling tags, like we avoided the `latest` tag.

Adding to `default.env` will allow you to override that tag for testing a newer version easier, if needed.

Reply:
I see what you mean now. 
I can add the variable to allow more specific version overrides, but I believe those patch version are often for security fixes that are better to obtain as soon as possible.
Note that all other dbs `mongo:3.4` and `postgres:9.6` (generic and magpie one) also use those rolling versions.

Reply:
> I believe those patch version are often for security fixes that are better to obtain as soon as possible.

The problem is docker do not automatically detect tag override and re-download them so we are falsely leading the user to believe they are getting rolling release update.

Also using rolling release means no reproductibility.

> Note that all other dbs `mongo:3.4` and `postgres:9.6` (generic and magpie one) also use those rolling versions.

Indeed, old code.  New code, we should be more disciplined.  Pretty sure in the past I've mistaken rolling tag for exact tag as well so it's something we have to be careful.

Not reproducible deployment means we can lose lots of time chasing false leads, like I've learned the hard way with Geoserver upgrade, see https://github.com/kartoza/docker-geoserver/issues/232